.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+###################################
+##### Global Protection Rule ######
+###################################
+# NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
+*                                               @hashgraph/hedera-smart-contracts
```

Comment:
I'm not sure why it's showing invalid team for `@hashgraph/hedera-smart-contracts`.

Here's an example from services
root dir - https://github.com/hashgraph/hedera-services/blob/develop/CODEOWNERS
github dir - https://github.com/hashgraph/hedera-services/blob/develop/.github/CODEOWNERS

Reply:
The team needs to have write access to the repository, it currently does not, and that's why it's showing invalid team.

Reply:
Thanks @isavov . I've fixed that now